_posts/2022-05-19-kotlin-context-receivers.markdown

```diff
@@ -0,0 +1,504 @@
+---
+layout: post
+title:  "Context Receivers in Kotlin"
+date:   2022-05-19 20:00:00 +0100
+categories: Kotlin
+author: Fatih Coşkun
+---
+# Context Receivers In Kotlin
+
+Kotlin [1.6.20](https://kotlinlang.org/docs/whatsnew1620.html) finally introduces 
+[context receivers](https://github.com/Kotlin/KEEP/blob/master/proposals/context-receivers.md#detailed-design), a new 
+language feature that subsumes several language features that have been hotly discussed on various Kotlin boards for
+many years.
+
+In this post we will see what context receivers are, how they compare to other types of receivers in Kotlin, and various
+use cases in which they solve problems more concisely than was possible before.
```

Comment:
Are we missing the 'how it' words here?
Instead of:
'in which they solve problems more concisely than was possible before'
should it be:
'in which they solve problems more concisely than how it was possible before'

Reply:
(y) I will change it